stubs/tqdm/tqdm/keras.pyi

```diff
@@ -1,10 +1,31 @@
 from _typeshed import Incomplete
-from typing import Any
-from typing_extensions import TypeAlias
 
 __all__ = ["TqdmCallback"]
 
-_Callback: TypeAlias = Any  # Actually tensorflow.keras.callbacks.Callback
+# keras.callbacks.Callback
```

Comment:
```suggestion
# tensorflow.python.keras.callbacks.Callback
```

Reply:
Weirdly enough, it's actually using keras, and if that fails, it fallsback to tensorflow... which uses keras (probably does some extra in that case idk)

Reply:
Oh good point, ignore me!